demo/index.md

```diff
@@ -205,6 +205,67 @@ It is possible to automatically sort the table once you create a Tablesort insta
 </tbody>
 </table>
 
+### Sorting by column keys
+
+Sometimes, tables can have more complex column structures, especially when using colspans. In these cases, you can
+explicitly connect a header to the cells in each row that it should sort by, by using the `data-sort-column-key`
+attribute.
+
+This example sorts products by price, even though the prices are not in the same column as their header.
+
+{% highlight html %}<table id='column-keys' class='sort'>
+<thead>
+  <tr>
+    <th >Product</th>
+    <th colspan=2 data-sort-column-key="price">Price</th>
```

Comment:
`colspan=2` value should be quoted! `colspan="2"`

Reply:
Whoops! 🤦‍♂